bpf/windows/process_monitor.c

```diff
@@ -147,13 +151,15 @@ int ProcessMonitor(process_md_t *ctx)
 
 	} else if (ctx->operation == PROCESS_OPERATION_DELETE) {
 		struct process_exit_info_t process_exit_info;
+		int size = sizeof(process_exit_info);
 
-		memset(&process_exit_info, 0, sizeof(process_exit_info));
+		memset(&process_exit_info, 0, size);
 		process_exit_info.process_id = ctx->process_id;
+		process_exit_info.common.op = MSG_OP_EXIT;
 		process_exit_info.common.ktime = ctx->exit_time;
+		process_exit_info.common.size = size;
```

Comment:
btw might be okay to reuse `sizeof(static)` in multiple places since in this case it should be evaluated at compile time. I imagine the compiler will optimize the size variable away!